testdata/testdata.go

```diff
@@ -157,6 +157,15 @@ func notUsed_IfStmt_Body_OK() {
 	}
 }
 
+func varUsedInSliceIndex_OK() {
+	index := getInt()
+	if index == 0 {
+		noOp1(index)
+	}
+	mySlice := []int{0}
+	noOp1(mySlice[getInt(index)])
```

Comment:
The library works correctly when you have `mySlice[index]`. Apparently it doesn't detect passing it to a function for indexing. I'm assuming it's not going deep enough in the expression.

Reply:
Maybe this should be called `varUsedInExpressionForIndexing_OK`